inspirehep/config.py

```diff
@@ -889,3 +894,5 @@ def _(x):
     'stat.OT': 'Other',
     'stat.TH': 'Other'
 }
+
+ASSETS_DEBUG = True
```

Comment:
Is this a leftover? We have been usually adding it in our instance folder local config, but maybe it makes sense to have it here?